core/consensus/common/src/import_queue.rs

```diff
@@ -565,7 +567,12 @@ pub fn import_single_block<B: BlockT, V: Verifier<B>>(
 			BlockImportError::VerificationFailed(peer, msg)
 		})?;
 
-	import_error(import_handle.import_block(import_block, new_authorities))
+	let mut cache = HashMap::new();
+	if let Some(authorities) = new_authorities {
+		cache.insert(b"auth".to_vec(), authorities.encode());
```

Comment:
I suppose it makes sense to move the constant somewhere.

Reply:
Suggested to use it in `core/consensus/aura/src/lib.rs`, so probably make something similar to `well_known_keys` (i.e. `well_known_cache_keys` in `substrate-consensus-common`)?